site/_includes/partials/devsite-header.njk

```diff
@@ -0,0 +1,39 @@
+{% from 'macros/icon.njk' import icon with context %}
```

Comment:
I think if we use the new header across all of the site we should also get rid of the sidebar everywhere and use the exact same header on every page. This would also fix the content jump when you navigate away from the homepage, as the other header is slightly smaller.